Omise.Net/RequestManager.cs

```diff
@@ -13,6 +13,7 @@ public class RequestManager : IRequestManager
         private string apiUrlBase;
         private string apiKey;
         private string encodedCredentials;
+        private readonly string clientVersion = "1.0.5";

```

Comment:
We could read from the assembly (.dll file) version setting but that means we need to reset and manually set the assembly version since now it is auto generated (e.g. 1.0.51231).